calico/maintenance/kubernetes-upgrade.md

```diff
@@ -68,6 +68,12 @@ ownership of the helm resources to the new chart location.
 
 ### All other upgrades
 
+1. Apply the {{page.version}} CRDs:
+
+   ```bash
+   kubectl apply -f {{site.data.versions.first.manifests_url}}/manifests/operator-crds.yaml
```

Comment:
I think this needs to be a `kubectl replace -f`

Reply:
```suggestion
   kubectl replace -f {{site.data.versions.first.manifests_url}}/manifests/operator-crds.yaml
```